distribution/oceanbase/build.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
- docker build -t $1 --build-arg GOPROXY=${GOPROXY} .
+docker build -t $1 --build-arg GOPROXY=$(go env GOPROXY) --build-arg VERSION=$2 .
```

Comment:
This requires go environment locally, it's not necessary for building docker image, just use the environment variable maybe a better choice

Reply:
OK, I'll put it back